### PR TITLE
Ember CLI 2016-09-22

### PR DESCRIPTION
**Pull request by nathanhammond**
Comments for corrections, reactions for approval.

/cc @stefanpenner @rwjblue @chadhietala @hjdivad @kellyselden @kategengler @trabus @Turbo87

### CODE REVIEW

File: ember-cli/2016-09/september-22.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- Robert Jackson
+- Tobias Bieniek
+- Katie Gengler
+- Stefan Penner
+- Chad Hietala
+- David Hamilton
+ 
+## Actions
+
+- [ ] Tobias: Open issue for making sourcemaps enable-able with warning.
```
Comment (stefanpenner) [inline ember-cli/2016-09/september-22.md]: 👍

File: ember-cli/2016-09/september-22.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Need to summarize:
+  - What's keeping us from having that.
+    - Babel gets slower.
+    - Sourcemap concat gets extremely slow.
+  - what's needed to make it happen
+   - how can new contributors help
+ 
+- Create a meta issue summarizing these things
+- Notes:
+  - Babel doesn't do sourcemaps correctly for symbol mangling in V5, maybe fixed in v6.
```
Comment (stefanpenner) [inline ember-cli/2016-09/september-22.md]: did we confirm babel 6 still doesn't?

Comment (nathanhammond) [inline ember-cli/2016-09/september-22.md]: @krisselden Do you know about this?

File: ember-cli/2016-09/september-22.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  - source map segments or sections? (confirm with @hjdivad) one doesn't need to `cat` sourcemaps.
+ 
+ 
+### Macro Blueprint
+
+- [Macro Blueprint RFC](https://github.com/ember-cli/rfcs/pull/67)
+- Implement as an addon in parallel with RFC providing its own blueprints.
+- Whether to include by default is the conversation of record.
+- Including the test infrastructure for a macro makes blueprint more valuable.
+- `utils/macros` as a home, push conversation into Ember Core team.
```
Comment (locks) [inline ember-cli/2016-09/september-22.md]: 🤔

**Pull request merged**